mmdet/datasets/dataset_wrappers.py

```diff
@@ -400,3 +404,92 @@ def update_dynamic_scale(self, dynamic_scale):
         """
         assert isinstance(dynamic_scale, tuple)
         self._dynamic_scale = dynamic_scale
+
+
+@DATASETS.register_module()
+class ImageTilingDataset:
```

Comment:
Can't these rapper datasets be implemented directly in OTE repo (external/mmdetection)?

Reply:
yes, we could move this to the OTX repo. That said, we don't have a dataset wrapper in OTX and I'm not sure where to store it as the replacement is still a work in progress. So personally it makes sense to put it in the mmdet dataset wrapper. Please let me know if you think we should be move it to OTX repo.